src/main/java/com/server/SimpleGet.java

```diff
@@ -31,20 +31,15 @@ public void runServer() throws IOException {
             String path = requestHeaderParser.getPath();
             int responseCode = requestRouter.getResponseCode(path, method);
 
-            String optionsLine = "";
+            ResponseHeaderBuilder responseHeaderBuilder = new ResponseHeaderBuilder(responseCode);
+
             if (method.equals("OPTIONS")) {
```

Comment:
This conditional is likely to change soon, but until we have more information, we'll leave it as is.